test/suites/edgexfoundry/secrets_config_test.go

```diff
@@ -41,3 +44,30 @@ func TestAddAPIGatewayUser(t *testing.T) {
 	require.NoError(t, err)
 	t.Logf("Output: %s", body)
 }
+
+// TestChangeTLSCert creats new TLS certificate and calls API gateway to verify the use of new certificate
+// https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSnapUsers/#changing-tls-certificates
+func TestChangeTLSCert(t *testing.T) {
+	t.Log("Create TLS certificate")
+	createTLSCert(t)
+
+	t.Log("Calling API gateway using new TLS certificates:", coreDataPingEndpoint)
+	utils.WaitServiceOnline(t, 60, utils.ServicePort("nginx(https)"))
+
+	const caCertFile = "./ca.cert"
+	// Note: %%	is a literal percent sign
+	// Note: The path to the ca.cert file is created by create-tls-certificates.sh
+	code, _, _ := utils.Exec(t, fmt.Sprintf(
+		"curl --show-error --silent --include --output /dev/null --write-out '%%{http_code}' --caCert %s '%s'",
```

Comment:
```suggestion
	// Note: The ca.cert file is created by create-tls-certificates.sh
	code, _, _ := utils.Exec(t, fmt.Sprintf(
		"curl --verbose --show-error --silent --include --output /dev/null --write-out '%%{http_code}' --cacert %s '%s'",
```
Argument is wrong.
Verbose flag helps with debugging.